### PR TITLE
Fix/move dep

### PR DESCRIPTION
**Pull request by itabrl**

### CODE REVIEW

**Comment by Supraluminal** [comment]
Circle CI is broken for this PR due to dependencies being moved into CARMABase, once CARMABase is released this will be fixed.

File: Dockerfile [hunk 1]
```diff
@@ -52,6 +52,33 @@ RUN apt-get update && \
         unzip \
         gcovr
 
+# Add dependencies for other CARMA package
+RUN apt-get install -y \
```
Comment (Supraluminal) [inline Dockerfile]: I think for ROS packages where possible we'd like to have them tracked in the `package.xml` for CARMABase. Then when we run `rosdep install...` during the CARMABase images install process, it finds them that way. I see some of these (like libpcap-dev) aren't rospackages so they're fine to stay here.

Comment (qswawrq) [reply Dockerfile]: I do not even realize we have a package.xml here! That makes sense to me. I can those updates on Monday.

**Comment by maefromm** [comment]
Looks like this is related to the dependencies issues, correct? If yes, please go ahead and merge. @itabrl and I will have to test all of these changes tomorrow.

**Pull request merged**